templates/blog/post_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+  
+  <h1>{{ post.title }}</h1>
+  <p>Автор: {{ post.owner.username }}</p>
+  <p>Створено: {{ post.created_time|date:"d.m.Y H:i" }}</p>
+  <p>{{ post.content }}</p>
+
+  <h2>Коментарі ({{ post.comment_count }})<a style="float: right" href="{% url 'blog:create' post.id %}">+</a></h2>
+  <ul>
+    {% for comment in post.commentary_set.all %}
```

Comment:
The loop is iterating over 'post.commentary_set.all'. Ensure that 'commentary_set' is the correct related name for comments associated with the post. If the related name is different, this will not work as expected.